### ISSUE TITLE
can't install full references

### ISSUE THREAD

**Issue by shinyaohira** [vvakame/dtsm#2]
my dtsm.json:
```json
{
  "repos": [
    {
      "url": "https://github.com/borisyankov/DefinitelyTyped.git",
      "ref": "master"
    }
  ],
  "path": "typings",
  "bundle": "typings/bundle.d.ts",
  "dependencies": {
    "rx/rx.all.d.ts": {
      "repo": "https://github.com/Reactive-Extensions/RxJS.git",
      "ref": "d376240a934f0d5bb1e4572161fd77ab6c6de57f",
      "path": "ts/rx.all.d.ts"
    }
  }
}
```

the result:
```
$ ls typings/rx
rx.aggregates.d.ts
rx.backpressure.d.ts
rx.d.ts
rx.testing.d.ts
rx.all.d.ts
rx.binding.d.ts
rx.experimental.d.ts
rx.time.d.ts
rx.async.d.ts
rx.coincidence.d.ts
rx.joinpatterns.d.ts
rx.virtualtime.d.ts
```

some files such as rx.async-lite.d.ts are not installed.

**Comment by vvakame** [comment]
@shinyaohira thanks! I'm fix it and release dtsm 0.1.1.

**Comment by shinyaohira** [comment]
Great!
Thanks :smile:
